src/components/FieldImage.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import Dropzone from 'react-dropzone';
+import { Field } from 'redux-form';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { getTranslator } from '../reducers/translation';
+import styles from './FieldImage.scss';
+
+const fileInstance = new FileReader(); // eslint-disable-line no-undef
```

Comment:
A better way to solve it is by putting `/* global FileReader */` at the top before the imports.